### PR TITLE
Add TODOs

### PR DESCRIPTION
**Pull request by balamurugana**

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* two types: fast test and full test. fast test is default.
```
Comment (NitishT) [inline TODO.md]: @balamurugana we have `core` tests, `stress` tests and `bench` as categories currently. `core` is the default test and all the current tests are under `core`. Other category tests are not added yet. (https://github.com/minio/mint/blob/master/tests.sh#L153)

IMO,we can add a category `full` tests for long running tests like unknown sized input stream multipart.

File: TODO.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+* tests should generate data its own by using input/output streams those streams should be used.  This removes /mint/data directory.
```
Comment (NitishT) [inline TODO.md]: Currently the `/mint/data` directory is [created only](https://github.com/minio/mint/blob/master/build/data/install.sh#L26) if it is not passed from outside while running the container. A volume can be passed from outside like this: https://docs.docker.com/edge/engine/reference/run/#volume-shared-filesystems

This makes it flexible for both who want to specify their data directory or those who want to run for default data created by mint. I am not clear why do we need to remove the /mint/data directory.

Comment (harshavardhana) [inline TODO.md]: This is incorrect generating data inside container is too slow. This is the whole reason to have pre-generated data? what benefit does it provide for tests themselves to generate data?

File: TODO.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+* unknown sized input stream of multipart test is required which is full test catogery.
```
Comment (harshavardhana) [inline TODO.md]: This is not a major requirement, if needed it can be provided by wrapping around an inputStream on a local file.

**Comment by harshavardhana** [comment]
@balamurugana can you close this if no longer relevant?

**Comment by balamurugana** [comment]
> @balamurugana can you close this if no longer relevant?

Once @NitishT opened respective issues in mint/SDKs. this can be closed.

**Comment by NitishT** [comment]
Created issues in respective SDKs

- https://github.com/minio/minio-java/issues/585
- https://github.com/minio/minio-py/issues/548
- https://github.com/minio/minio-js/issues/602
- https://github.com/minio/minio-go/issues/742
- https://github.com/minio/minio-dotnet/issues/147

**Comment by NitishT** [comment]
Added relevant issues. closing this

**Pull request closed without merging**